src/main/resources/META-INF/resources/webjars/form-flow/0.0.1/js/formFlowHideField.js

```diff
@@ -0,0 +1,13 @@
+function toggleVisibility(checkbox, selector) {
+  let element = document.querySelector(selector);
+  if (element) {
+    element.style.display = checkbox.checked ? 'none' : 'block';
```

Comment:
also, let's toggle the class "hidden" - that is basically setting "display: none"